scout/models/case/case.py

```diff
@@ -79,8 +80,8 @@
     phenotype_terms=list,  # List of dictionaries with phenotype information
     phenotype_groups=list,  # List of dictionaries with phenotype information
     madeline_info=str,  # madeline info is a full xml file
-    chromograph_image_files=str,  # path to dir
-    chromograph_prefixes=list,  # path to dir
+    chromograph_image_files=str,  # path to dir  # TODO: still needed?
```

Comment:
TODO.. :wink:

Reply:
Removing this.

Tried to think and think if removing it would cause any problems. As it wasn't used shouldn't affect anything.

Reply:
Revisiting an old PR (https://github.com/Clinical-Genomics/scout/pull/1999) I conclude that the dicts constructed in `case/case.py` never come to use. Removing it will be safe.